minecraft/protocol/packet/npc_dialogue.go

```diff
@@ -0,0 +1,49 @@
+package packet
+
+import "github.com/sandertv/gophertunnel/minecraft/protocol"
+
+const (
+	NPCDialogueActionTypeOpen int32 = iota
+	NPCDialogueActionTypeClose
```

Comment:
Would remove `Type` here to make it slightly shorter.

Reply:
Removed.